webapp/package.json

```diff
@@ -11,26 +11,29 @@
   "author": "",
   "license": "",
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
-    "eslint": "^5.2.0",
-    "eslint-import-resolver-webpack": "^0.10.1",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-react": "^7.10.0",
-    "file-loader": "3.0.1",
-    "webpack": "^4.16.1",
-    "webpack-cli": "^3.0.8"
+    "@babel/cli": "7.4.4",
```

Comment:
This wasn't required previously. Why is it now?

Reply:
I think they split the package out in the upgrade to 7.

Reply:
:ok: